CODE_REVIEW.md

```diff
@@ -0,0 +1,45 @@
+## Code Review ##
+
+### Issues ###
+
+* BookSearchComponent: When you search for a new book and the search result panel opens, the previous search results are not cleared and are visible.
+
+* BookSearchComponent: There is not much error handling if the bookSearch API call fails - the user would not know if an error occured. This could be remedied by using the BookState attribues 'loaded' and 'error'.
+
+### Points of Improvement ###
+
+* In terms of functionality, I would add a button to clear the search input. As of now, you need to manually backspace the input if you want to clear the field.
+    * I added this feature
+
+* BookSearchComponent: I noticed there is a difference in how we are subscribing to data store changes. In this component, we are:
```

Comment:
...and? Noticing the difference is one thing. What does the difference _mean_?

I'll give you a hint, it has to do with memory, subscriptions and the async pipe.

Reply:
Thanks for the pointer. I've updated Code_Review with a detailed description